docs/node.md

```diff
@@ -62,28 +62,30 @@ After creation, Kubernetes will check whether the node is valid or not.
 For example, if you try to create a node from the following content:
 ```json
 {
-  "id": "10.1.2.3",
-  "kind": "Minion",
-  "apiVersion": "v1beta1",
-  "resources": {
-    "capacity": {
-      "cpu": 1000,
-      "memory": 1073741824
-    },
-  },
-  "labels": {
-    "name": "my-first-k8s-node",
+  "kind": "Node",
+  "apiVersion": "v1beta3",
+  "metadata": {
+    "name": "10.240.79.157",
+    "labels": {
+      "name": "my-first-k8s-node"
+    }
   },
+  "status": {
```

Comment:
Should you create a node with pre-set capacity? kubelet reports node status and it will use the machine info from cadvisor to fill the values. I am not sure a user should provide the status directly.